[Known Issues] Add Elastic Managed LLM known issue (#227514)
elastic/kibana: docs/CHANGELOG.asciidoc

====
// end::known-issue-220339[]

// tag::known-issue-2088[]
.Observability AI Assistant - Elastic Managed LLM may be automatically selected as default connector
[%collapsible]
====
*Details* +

> The Elastic Managed LLM may be automatically be selected as your default connector because of existing connector selection logic. 

Author: ```suggestion
The Elastic Managed LLM may be automatically selected as your default connector because of existing connector selection logic. 
```

This may occur if you had not previously selected a specific connector for any of the following reasons:

- You only had one connector available and it was always automatically picked for your conversations.

> - You had multiple connectors available but didn’t make a specific selection and used the automatically picked connector for their conversations.

Author: ```suggestion
- You had multiple connectors available but didn’t make a specific selection and used the automatically picked connector for your conversations.
```

> - You previously selected a connector but cleared browser local storage or switched browsers/devices.

Author: ```suggestion
- You previously selected a connector but cleared your browser's local storage or switched browsers or devices.
```

> - All of your existing connector names sort alphabetically after “Elastic Managed LLM connector."

Author: ```suggestion
- All of your existing connector names come after “Elastic Managed LLM connector" when sorted alphabetically.
```

> - All of your existing connector names come after “Elastic Managed LLM connector" when sorted alphabetically.

Reviewer: @mdbirnstiehl 
The customers will face this issue if they fall under one of the following 3 reasons

> - You only had one connector available and it was always automatically picked for your conversations.
> - You had multiple connectors available but didn’t make a specific selection and used the automatically picked connector for your conversations.
> - You previously selected a connector but cleared your browser's local storage or switched browsers or devices.

AND "All of your existing connector names come after “Elastic Managed LLM connector" when sorted alphabetically"

Does it make sense?

Because even if they have the one of the first 3 reasons mentioned, but their custom connector starts with a letter before "E" (example: "ACME connector"), then the Elastic Managed LLM won't be their default connector.

Reviewer 2: It does, suggested an edit below

For more information, check https://github.com/elastic/docs-content/issues/2088[#2088]
====

> // end::know-issue-2088[]

Author: ```suggestion
// end::known-issue-2088[]
```
